lnbits/app.py

```diff
@@ -52,47 +50,25 @@ def create_app(config_object="lnbits.settings") -> FastAPI:
             "url": "https://raw.githubusercontent.com/lnbits/lnbits-legend/main/LICENSE",
         },
     )
+
     app.mount("/static", StaticFiles(packages=[("lnbits", "static")]), name="static")
     app.mount(
         "/core/static",
         StaticFiles(packages=[("lnbits.core", "static")]),
         name="core_static",
     )
 
-    origins = ["*"]
+    # needed for lnurlw?
+    # g().config = settings
+    g().base_url = f"http://{settings.host}:{settings.port}"
```

Comment:
What is this for? Why http? LNbits can run in https.

Reply:
This is probably leftovers from first iteration before @dni 's great refactor! I was putting the settings in the `g()` global object... maybe it can be removed @dni ?

Reply:
i have no idea what the g() is aswell, i would like to get rid of it. but i noticed uncommented `g().base_url = f"http://{settings.host}:{settings.port}"`break lnurlp or lnurlw

Reply:
g() is also used on main branch, so ill keep it, i just rmeoved the settings part.